components/dialogs/gatekeeper-management-dialog.tsx

```diff
@@ -139,6 +146,19 @@ export function GatekeeperManagementDialog({
   const isStandalone = useMediaQuery("(display-mode: standalone)");
   const t = useTranslations("gatekeeper-management-dialog");
 
+  const { data: communitiesPages } = useSuspenseInfiniteQuery(
+    communitiesListByEvent(eventId, DEFAULT_COMMUNITIES_LIMIT),
+  );
+  const communities = useMemo(
+    () => communitiesPages.pages.flat(),
+    [communitiesPages],
+  );
+
+  const communityId =
+    communities.length > 0
+      ? communityIdFromPkgPath(communities[0].pkgPath)
+      : null;
```

Comment:
why we need this in gatekeeper management dialog ? is it because its to call edit event so we fill with default values ?